### PR TITLE
Add readme templates

### PR DESCRIPTION
**Pull request by mzorn-58**

### CODE REVIEW

File: model-output/README.md [hunk 1]
```diff
@@ -1,3 +1,331 @@
...
+
+    YYYY-MM-DD
+    
+This is the last date of the forecast target's week. This will be the date of the Saturday at the end of the forecasted week. As a reminder, the `target_end_date` is the end date of the week during which the admissions are reported (see the data processing section for more details). Within each row of the submission file, the `target_end_date` should be equal to the `reference_date` + `horizon`*(7 days).
+
+
+### `location`
+
+Values in the `location` column must be one of the "locations" in
+this [location information file](../auxiliary-data/locations.csv) which
```
Comment (nickreich) [inline model-output/README.md:167]: is this file present in this repo?

File: model-output/README.md [hunk 2]
```diff
@@ -1,3 +1,331 @@
...
+Python: quantiles =
+np.append(np.append([0.01,0.025],np.arange(0.05,0.95+0.05,0.05)),
+[0.975,0.99])
+
+
+### `value`
+
+Values in the `value` column are non-negative numbers indicating the "quantile" or "pmf" prediction for this row. For a "quantile" prediction, `value` is the inverse of the cumulative distribution function (CDF) for the target, location, and quantile associated with that row. For example, the 2.5 and 97.5 quantiles for a given target and location should capture 95% of the predicted values and correspond to the central 95% Prediction Interval. 
+
+### Example tables
```
Comment (nickreich) [inline model-output/README.md:209]: maybe this level of FLuSight-specific detail does not need to be included here in this bare-bones template?

File: README.md [hunk 3]
```diff
@@ -1,7 +1,36 @@
...
-**This repository is in development.**
-
-This repository serves as a template for groups wishing to build and maintain a new modeling hub, in the style of the [US COVID-19 Forecast Hub](https://github.com/reichlab/covid19-forecast-hub), the [US Scenario Modeling Hub](https://github.com/midas-network/covid19-scenario-modeling-hub) or the [European COVID-19 Forecast Hub](https://github.com/covid19-forecast-hub-europe/covid19-forecast-hub-europe). Researchers from these teams have created the Consortium of Infectious Disease Modeling Hubs to provide a [recommended set of standards and documentation](https://hubdocs.readthedocs.io/en/latest/index.html) for building and maintaining collaborative modeling hubs.
-
-This repository may be cloned to start a new repository for a modeling hub.
\ No newline at end of file
+<mark style="background-color: #FFE331">**This README.md is from [Flusight-Forecast_Hub  README](https://github.com/cdcepi/FluSight-forecast-hub/blob/master/README.md).  Be sure to modify the text below to reflect your hub.**</mark>
+
+
+# FluSight 2023-2024
```
Comment (nickreich) [inline README.md:4]: I think in general this is too much FluSight-specific detail for a repository that is designed to be just a template.

**Comment by mzorn-58** [comment]
I removed Flusight content and provided a link to the Flusight readme.md at the bottom of the templates.

**Review by nickreich — APPROVED** [review]

**Pull request merged**